sdk/language-support/ts/daml-ledger/index.ts

```diff
@@ -260,7 +270,8 @@ async function decodeArchiveResponse<T extends object, K, I extends string>(
   if (
     events.length === 1 &&
     "archived" in events[0] &&
-    events[0].archived.templateId === template.templateId
+    stripPackage(events[0].archived.templateId) ===
+      stripPackage(template.templateId)
```

Comment:
Again, we ignore the package specifier here, because the request can contain a package name while the response will contain a package id.